styles/style.css

```
@@ -0,0 +1,491 @@
+  color: #666;
+  margin: 4px 0;
+  padding: 4px 0;
+}
+
+ .poster-subsection {
+  display: flow-root; 
+}
+
+ .poster-article {
+  float: left;
+  font-family: nyt-franklin,arial,helvetica,sans-serif;
+  font-size: 13px;
+  font-weight: 300;
+  width: 230px;
+  margin: 7px;
+  color: #000000;
+  font-style: normal;
+}
+
+ .poster-article > img {
+  display: block;
+  width: 228px;
+  height: 120px;
+}
+
+.part2 {
+  padding-bottom: 30px;
+}
+
+#poster-footer {
+  clear: both;
+  font-family: nyt-franklin,arial,helvetica,sans-serif;
+  font-size: 13px;
+  font-weight: lighter;
+}
+
+#poster-footer p::before {
+  content: 'By LARRY ';
+  border-top: 1px solid black;
+  padding-top: 20px;
+}
+
+a#celebration {
+  color: #326891;
+}
+
+#youtube-video {
+  padding: 20px 0;
+  color: #666;
+}
+
+#youtube-video > p {
+  font-size: 15px;
+  font-weight: 400;
+  color: #666;
+  margin: 2px 0;
+  padding: 2px 0;
+}
+
+#youtube-video > span {
+  display: block;
+  font-size: 10px;
+}
+
+#bicep2 {
+  display: block;
+  width: 600px;
+  height: 401px;
+}
+
+#bicep-t > p {
+  font-size: 15px;
+  font-weight: 400;
+  color: #666;
+  margin: 2px 0;
+  padding: 2px 0;
+}
+
+#bicep-t > span {
+  display: block;
+  font-size: 10px;
+}
+
+.sections > small {
+  margin: 0 auto 1em;
+  color: #999;
+  font-size: 0.75rem;
+  text-decoration: none;
+}
+
+a.links {
+  color: #999;
+}
+
+#comment {
+  float: left;
+  background-color: rgb(98, 136, 165);
+  border: 1px solid rgb(50, 104, 145);
+  border-radius: 3px;
+  color: white;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 0.8125rem;
+  font-weight: 600;
+  width: 330px;
+  height: 30px;
+  margin-top: 5px;
+}
+
+#article-footer {
+  margin: auto;
+  display: block;
+  padding-left: 30px;
+  width: 1500px;
+  justify-content: center;
+  padding-bottom: 950px;
+}
+
+#leftside {
+  width: 70%;
+  float: left;
+  margin: 30px 40px 30px 10px;
+  border-top: 1px solid black;
+  width: 822px; 
+}
+
+#rightside {
+  width: 30%;
+  float: right;
+  margin: 30px 30px 60px 10px;
+  border-top: 1px solid black;
+  width: 400px;
+}
+
+#first-row {
+  display: flow-root;
+}
+
+.row-block {
+  float: left;
+  height: 230px;
+  width: 264px;
+  margin: 0 5px;
+}  
+
+.row-block > img {
+  height: 174px;
+  width: 262px;
+}
+
+.row-block > p {
+  padding: 0;
+  margin: 0;
+  font-size: 10px;
+  font-family: nyt-cheltenham-small, georgia, "times new roman";
+  text-align: right;
+  color: rgb(153, 153, 153);
+}
+
+.row-block h2 {
+  font-size: 15px;
+  font-family: nyt-cheltenham,georgia,"times new roman",times,serif;
+}
+
+.row-block small {
+  display: block;
+  color: rgb(153, 153, 153);
+  margin-top: -6px;
+}
+
+.UBS {
+  font-family: 'Source Sans Pro',sans-serif;
+  background-color: #F6F6F6;
+}
+
+#USB-content {
+  font-size: 12px;
+  padding: 8px 5px;
+  font-weight: bolder;
+  color: #121212;
+  background-color:  #F6F6F6;
+}
+
+#UBS-logo {
+  height: 30px;;
+  width: 264px   
```

Let's add the missing semicolon(;), this property is currently invalid without it.

```suggestion
  width: 264px;
```
###### There's also an extra semicolon in the previous property (`height`), let's remove that.